chart/openfaas/README.md

```diff
@@ -67,12 +56,19 @@ Now deploy OpenFaaS from the helm chart repo:
 helm repo update \
  && helm upgrade openfaas --install openfaas/openfaas \
     --namespace openfaas  \
-    --set basic_auth=true \
-    --set functionNamespace=openfaas-fn
+    --set functionNamespace=openfaas-fn \
+    --set generateBasicAuth=true 
 ```
 
 > The above command will also update your helm repo to pull in any new releases.
 
+Retrieve the OpenFaaS credentials with:
+
+```sh
+PASSWORD=$(kubectl -n openfaas get secret basic-auth -o jsonpath="{.data.basic-auth-password}" | base64 --decode) && \
```

Comment:
Can we also add this command to the `NOTES.txt` output when the `generateBasicAuth` is true?  That way it is really easy to find in the terminal right after install

Reply:
Done

Reply:
Done

Reply:
Awesome